data/persons.txt

```diff
@@ -2392,3 +2392,45 @@ ship "Protector" "Modified Protector"
 	explode "large explosion" 8
 	"final explode" "final explosion large"
 	description `Voted the "ugliest ship in the sky" by Stars and Starships Magazine, the Protector is a typical example of brutally efficient Syndicate engineering. It is basically nothing more than eight turrets attached to a set of engines and crew's quarters, designed as a defense platform that can accompany merchant convoys.`
+
+
+
+person "Sanujab"
+	government "Hai"
+	frequency 100
+	personality
+		disables harvests timid
+	system
+		government "Hai" "Quarg (Hai)"
+	phrase
+		word
+			"Hey there!"
+			"Vai bin so."
+			"You're a human, right?"
+			"Remember, only those of peace are allowed in Hai space." 2
+			"I've never owned a Shield Beetle, and have got no intention of ever upgrading." 2
+			"Do you know what time it is on Earth?"
+			"Do you know what time it is on Hai-home?"
+			"Nobody knows how I think. Not even myself." 2
+			"I don't like fighting the Unfettered. Their Trackers are a nuisance." 2
+			"They say Quantum Keystones make you lucky. Guess I'm in luck." 2
```

Comment:
```suggestion
			"Remember, only those of peace are allowed in Hai space."
			"I've never owned a Shield Beetle, and have got no intention of ever upgrading."
			"Do you know what time it is on Earth?"
			"Do you know what time it is on Hai-home?"
			"Nobody knows how I think. Not even myself."
			"I don't like fighting the Unfettered. Their Trackers are a nuisance."
			"They say Quantum Keystones make you lucky. Guess I'm in luck."
```

Is there a purpose for the "2" on the end of several lines?

Reply:
That is... weird that got sent before I sent the full review. 

In any case, thanks for the explanation. I'm good with that, then. No need to apply my suggestion.

Reply:
```suggestion
			"Remember, only those of peace are allowed in Hai space."
			"I've never owned a Shield Beetle, and have got no intention of ever upgrading."
			"Nobody knows how I think. Not even myself."
			"I don't like fighting the Unfettered. Their Trackers are a nuisance."
			"They say Quantum Keystones make you lucky. Guess I'm in luck."
			"Do you know what time it is on ${deep planets}?" 4
			"Better get a pen and pencil ready, I'm about to ask a whole bunch of questions!"
			"Did you know that we Hai already had a mountain named ${mountains} before we met you?"
			"I like saying random words. ${civilian} ${civilian}."
			"Have you ever seen a ship named ${civilian}? It's an interesting name."
			"I can't decide what planet I'm from. Maybe ${hai planets}?"
```

Reply:
Hmmm, not sure I'm a fan of the "I like saying random words" line, but the other additions look good. Maybe decrease the frequency of the "do you know what time it is" one from 4 down to 2. I think that should be enough.